src/main/kotlin/com/playwright/remote/engine/download/impl/Download.kt

```diff
@@ -0,0 +1,55 @@
+package com.playwright.remote.engine.download.impl
+
+import com.google.gson.JsonObject
+import com.playwright.remote.engine.browser.api.IBrowser
+import com.playwright.remote.engine.browser.api.IBrowserContext
+import com.playwright.remote.engine.download.api.IDownload
+import com.playwright.remote.engine.download.stream.api.IStream
+import com.playwright.remote.engine.processor.ChannelOwner
+import com.playwright.remote.utils.Utils.Companion.writeToFile
+import java.io.InputStream
+import java.nio.file.Path
+
+class Download(parent: ChannelOwner, type: String, guid: String, initializer: JsonObject) : ChannelOwner(
+    parent,
+    type,
+    guid,
+    initializer
+), IDownload {
+    private var browser: IBrowser = (parent as IBrowserContext).browser()
```

Comment:
This property of Download class is not used inside the class.
Remove it or make use of it.